mmv1/third_party/terraform/services/clouddeploy/resource_clouddeploy_custom_target_type_test.go

```diff
@@ -80,6 +80,14 @@ resource "google_clouddeploy_custom_target_type" "custom-target-type" {
           ref = "main"
         }
       }
+      include_skaffold_modules {
+        configs = ["my-config3"]
+        google_cloud_build_repo {
+          repository = "projects/777/locations/us-central1/connections/git/repositories/example"
```

Comment:
Where else is this repository being used? Or is this only being used for running the tests that you included in the PR?

Reply:
This repository does not exist and does not have to exist - it is only being used for running the tests in this PR.